### DISCUSSION TITLE
Can GROOPS do PPP using BeiDou?

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#76] [Q&A] [answered]
### Description

Hello developers,
  I'd like to do PPP with Beidou constellation, but the results are not good. Here are results of fix ambiguity and float ambiguity in ppp processing(note that doy: 2020 002 station:graz products:wum):
 
![图片1](https://user-images.githubusercontent.com/59271400/178107324-3d57fec1-85ed-4ede-85bf-74bf37ac1df1.jpg)
Figure 1 fix ambiguity
![图片1](https://user-images.githubusercontent.com/59271400/178107194-eb26b918-4f9a-4459-ba10-5e5c4e55e89e.jpg)
Figure 2 float ambituity
Note that only band L2 and L6 are used, and I there aren't any problem when I do ppp with GPS or Galileo constellation. By the way, I'v met the same problem when I do ppp with Glonass constellation as Beidou. Could you give me some suggestion about that? 
Regards, 
I'm looking forward to your reply!  
  

### GROOPS version

main (latest commit)

### Operating systems

- [ ] Linux
- [X] Windows
- [ ] macOS
- [ ] Other (please specify in the description)

**Comment by Waley04** [comment]
Hi @FFFFFG-IGG,

GROOPS Developers Can Explain Better. 

I tried a PPP test with the following parameters:

== DOY        : 002 in 2020
== Products : WUM0MGXRAP *
== Station    : jfng and graz
== System    : BDS only
== Freq        : B1 and B3

###############################################

FIXED SOLUTION

1-JFNG Station ---

![position_2020-01-02 jfng](https://user-images.githubusercontent.com/101952777/178162057-bdef44f9-410b-4d63-9c20-b4ada6f5b534.png)

2-GRAZ Station ---

![position_2020-01-02 graz](https://user-images.githubusercontent.com/101952777/178163271-0363cc20-a364-4616-a15e-98f60454aeab.png)


###############################################

FLOAT SOLUTION

1-JFNG Station ---

![position_2020-01-02 jfng](https://user-images.githubusercontent.com/101952777/178162102-515225c9-6369-4445-9f73-04d2c8c59a89.png)

2-GRAZ Station ---

![position_2020-01-02 graz](https://user-images.githubusercontent.com/101952777/178162110-f0709b72-1f68-4a47-9883-3fb8d78b48b8.png)

Best wishes,

Waley.

###########################################################
Disclaimer: I just know how to deploy PPP experiments in this software.
GROOPS developers can explain the quality of the solution.
###########################################################

**Reply by FFFFFG-IGG** [reply]
Thanks for your kind response. I'd like to ask if you sovle this problem?

**Comment by zhedumi** [comment, marked as answer]
Hi,

Yes using BeiDou in PPP is possible but I should also note that we currently have a development branch for BeiDou with some updates, improvements and fixes for future releases. 

To the results of yours it is interesting that in your results the float solution seems to be doing better than the fixed solution which is definitly not what I would expect. 
Might I know what parametrization, models you both used to get these results and what observations, satellite products, clock-products, phase biases etc. you used?

On a sidenote GLONASS always gives worse results since the observations/orbits are usually worse in comparison to other GNSS. Check [Strasser 2022](https://pure.tugraz.at/ws/portalfiles/portal/40917089/Strasser_2022_Doctoral_Thesis_Published_Version.pdf) Chapter 8.2. 

Best regards,
Patrick

**Reply by FFFFFG-IGG** [reply]
Thanks for your reply!
Models and parametrization are the same as groops ppp document, except that I do not estimate STEC.
And obs, osb, obx , sp3, clk are downloaded from WHU, MGEX FTP, which is run by J.Geng, rest of the products are obtained from TUG, as the document(TUG example doc) of ppp says.
I wonder if you have got good ppp results using BDS in your modified groops? And, does that branch have been merged into the groops main branch?
THX.
